quiz/generic/main.php

```diff
@@ -10,14 +12,65 @@
 include "./quiz_page.inc"; // qp_round_id_for_pi_toolbox
 
 $header_args = [
+    "css_files" => [
+        "$code_url/styles/struct.css",
+        "https://cdn.jsdelivr.net/npm/quill@2.0.3/dist/quill.core.css",
```

Comment:
should we just serve js/css from our domain since our user base is very sensitive to external resources?

Reply:
Yes we should, It didn't seem very clear to me how to download the files so I took the easy way and had to change pinc/base.inc to make this work.

Reply:
Looks like the css nor js have any imports or external references, so should just be able to paste into our external resources folder (/pinc/3rdparty) with appropriate license and details.json file.